alg/teca_ar_rutz.py

```diff
@@ -26,20 +71,56 @@ def update(self):
         self.impl.update()
 
     def get_report_callback(self):
+        """ Returns a proper TECA report callback function """
         def report_callback(port, md_in):
-            # pass through by default
+            """ Defines the variables that this class returns. """
+            # add the names of the variables this method generates
+            md_in[0].append('variables', 'IVT')
+
             return md_in
+        
         return report_callback
 
     def get_request_callback(self):
+        """ Returns a proper TECA request callback function """
         def request_callback(port, md_in, req_in):
-            # pass the request through
+            """ Requests the variables needed to find ARs """
+            # add the name of arrays that we need to find ARs
+            req_in['arrays'] = ['QV','u','v','pressure','lat','lon']
```

Comment:
We probably will need to allow user to change the name of these variables. Pressure field may be stored as "pressure" in out simulation output and "PSL" in another.